lib/active_campaign/errors.rb

```diff
@@ -27,6 +27,14 @@ def initialize(response = nil, exception = nil)
     def message
       if response.nil?
         super
+      elsif response.is_a?(Exception)
```

Comment:
Would it make sense to be a little more specific? 

Also, there are a bunch of duplication now that code climate warns about. Would it perhaps make sense to address that? I am hesitant to merge something that doesn't pass the rubocop style guide.